TACC/Core-CMS
# GH-157: Change Standard Font (+ Some Polish)

# Overview

This is part of the [Frontera Homepage Redesign](https://github.com/TACC/Core-CMS/issues/157) that is being propagated to Core.

- Set and load new standard font.
- Move and cleanup font settings.

# Issues

- GH-157

# Changes

1. Update import because of a file move (see change 4a).
2. Add import and export stylesheet for loading font faces.
3. Load all Benton Sans font faces (that Frontera Redesign uses).
4. Move settings, change one value, cleanup file
    - Move font.css settings from site_shared to site_cms.*
    - Change global font family.
    - Cleanup font settings.
        - Update comments.
        - Comment out unused settings.
        - Add font weight settings (and shorthand)

* I am silently, gradually deprecating `site_shared`.

# Testing

This is already tested in context via https://frontera-portal.tacc.utexas.edu/home-2021-03/ or (if launched) https://frontera-portal.tacc.utexas.edu/ and locally by developer with other related changes via https://github.com/TACC/Core-CMS/pull/174.

1. Open a CMS page.
2. Ensure font family is "Benton Sans" for the body.
3. Add text on page via CMS Text plugin.
4. Confirm font family is "Benton Sans" for the text on the page.
